cypress/lib/signInOkta.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Non-mocked sign-in with Gateway using Okta
+ */
+
+export const signInOkta = () => {
+	// When this function runs, the browser will already be showing the Gateway sign-in page
+	// because MMA will have redirected to it when it loads the first page of the test.
+	cy.setCookie('gu-cmp-disabled', 'true', {
+		domain: '.thegulocal.com',
+	});
+
+	// Necessary otherwise we get a 502 error back from MMA for some reason, perhaps a race condition?
+	// TODO: Make this not suck
+	cy.wait(1000);
```

Comment:
Is there a smarter way to wait for a particular event?  I know that often with these kind of tests the answer is no.